tests/qgis-projects/tests/dnd_form.sql

```diff
@@ -65,7 +65,7 @@ ALTER TABLE ONLY tests_projects.dnd_form ALTER COLUMN id SET DEFAULT nextval('te
 --
 
 COPY tests_projects.dnd_form (id, field_in_dnd_form, field_not_in_dnd_form) FROM stdin;
-1	test	test
+1   test    test
```

Comment:
There is an error here because of the replacement of TAB by spaces -> the SQL does not work

Reply:
Do not forget to fix the other COPY statement

Reply:
> There is an error here because of the replacement of TAB by spaces -> the SQL does not work

So we should a test on the test ;-)

Reply:
Fix with `INSERT INTO ... VALUES`